docs/support.rst

```diff
@@ -7,9 +7,19 @@ Get support
 Get online support
 ==================
 
-(jitsi online meeting)
+There is a monthly online meeting, which you can join without registration.
+It takes place on every second Monday of a month at HH:MM CE(S)T. You can
 
 User forum
 ==========
 
-(openmod)
+If you have a question and you cannot attend an online meeting you can also
```

Comment:
I think, the forum is the first resource and if you prefer personal contact, we also have a consultation hour. (Forum posts can be answered asynchronously and answers can also help others.)